### DISCUSSION TITLE
Inventory saving options

### DISCUSSION THREAD

**Discussion by PovilasLT** [Zrips/CMI#8044] [Enhancement]
Hey,
Inventory saving function would be much more usable if it had few options:

- Don't save inventories in certain worlds (some servers like mine has Duels worlds, and after few rounds /invlist is full of crappy duels kit inventories)
- Add ability to save inventories on logging out and/or every x amount of playtime

**Comment by CloudeLecaw** [comment]
the secound one can be manged with EventCommands.yml
